packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -61,7 +89,29 @@ interface Props {
   queryRef: PreloadedQuery<ActivityLibraryQuery>
 }
 
-const getTemplateValue = (template: {name: string}) => template.name
+const CATEGORY_KEYWORDS: Partial<Record<CategoryID, string[]>> = {
+  strategy: ['okrs', 'goalsetting', 'goal-setting', 'planning'],
+  estimation: ['sprint'],
```

Comment:
Hmm I disagree. `poker` should still match most estimation templates since we're also matching against parabol meeting type. Additionally, if a team were to create a retro template to help them with an estimation discussion (e.g. a cost/benefit template), then `poker` wouldn't be a relevant search keyword for that template